tests/es6modules/jsfmt.spec.js

```diff
@@ -1,2 +1,13 @@
 run_spec(__dirname);
 run_spec(__dirname, { parser: 'babylon' });
+
+/**
+ * TS currently broken
+ *
+ * - export_default_function_declaration_async.js
+ *
+ * DIFF:
+ *   -export default (async function foo() {});
+ *   +export default async function foo() {}
```

Comment:
So how would we move forward with this? Just record the TS as an additional snapshot?